build.sh

```diff
@@ -18,4 +18,10 @@ sed -e "/#INSERT_PYTHON_CODE_HERE/r ./src/aws_cli_mfa.py" -e "s///" ./src/$AWS_C
     sudo chmod +x /usr/local/bin/aws-cli-mfa &&
     cp /usr/local/bin/aws-cli-mfa ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL
 
+sed 's/"/\\\"/g' ./src/aws_cli_mfa.py | sed -e "/#INSERT_PYTHON_CODE_HERE/r /dev/stdin" -e "s///" ./src/mfa.fish |
+    sudo tee /usr/local/bin/aws-cli-mfa >/dev/null &&
```

Comment:
Sourcing this file, even though it is on `$PATH`, does not work in fish (by design according to one of the committers in gitter).  To get around this one option would be to wrap it in a script that sources the file using its full path.

Reply:
That being said, I was able to get session tokens and make aws calls locally using the new fish scripts, I just had to use the full path for `aws-cli-mfa`.